scripts/eval_diar.py

```
@@ -0,0 +1,824 @@
+    recluster_entries: list[dict] = []
+    diar_time = 0.0
+    for group_idx, group in enumerate(groups):
+        g_start, g_end = group[0][0], group[-1][1]
+        buf = samples[g_start:g_end]
+        fast_labels = [lbl for _, _, lbl in group]
+        majority = max(set(fast_labels), key=fast_labels.count) if fast_labels else ""
+
+        t0 = time.time()
+        try:
+            raw = diarizer.process(buf, sr)
+        except Exception:
```

Ruff (0.16.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)